docs/Guides/Serverless.md

```diff
@@ -64,7 +64,7 @@ if (require.main === module) {
 
 When executed in your lambda function we do not need to listen to a specific
 port, so we just export the wrapper function `init` in this case. The
-[`lambda.js`](https://www.fastify.io/docs/latest/Serverless/#lambda-js) file
+[`lambda.js`](#lambdajs) file
```

Comment:
I'm pretty sure there was a reason for the full URL link, but I can't recall it.

Reply:
I thought so to, but couldn't find any